libs/blocks/gnav/gnav.js

```diff
@@ -334,11 +351,22 @@ class Gnav {
     });
     const searchResults = createTag('div', { class: 'gnav-search-results' });
 
+    // TODO: Fix locale URLs.
+    const href = new URL(advancedLink.href);
+    advancedLink.remove();
+
     searchInput.addEventListener('input', (e) => {
-      this.onSearchInput(e.target.value, searchResults, advancedLink);
+      this.onSearchInput(e.target.value, searchResults);
+    });
+
+    searchInput.addEventListener('keydown', (e) => {
+      if (e.keyCode === 13) {
```

Comment:
`keyCode` is deprecated, `code` should be used instead
```suggestion
      if (e.code === 'Enter') {
```